### PR TITLE
Correcting Liaison typos : Attempt 2

### PR DESCRIPTION
**Pull request by Ghommie**
It is just faster to close a PR and reopen it for such small frame changes.

### CODE REVIEW

File: maps/backup/Z.03.USS_Almayer.dmm [hunk 1]
```diff
@@ -865,7 +865,7 @@
 "aqG" = (/obj/effect/decal/warning_stripes{tag = "icon-S"; icon_state = "S"},/turf/open/floor/almayer{icon_state = "plate"; tag = "icon-sterile"},/area/almayer/hull/upper_hull)
 "aqH" = (/turf/open/floor/almayer{tag = "icon-red (SOUTHEAST)"; icon_state = "red"; dir = 6},/area/almayer/command/cic)
 "aqI" = (/obj/item/device/flashlight/lamp,/obj/structure/table/almayer,/obj/machinery/computer/security/almayer{dir = 4; layer = 4; name = "Ship Security Cameras"; network = list("almayer"); pixel_x = 17; pixel_y = 16},/turf/open/floor/almayer{icon_state = "plate"; tag = "icon-sterile"},/area/almayer/command/cic)
-"aqJ" = (/turf/open/floor/plating,/obj/effect/attach_point/weapon/dropship1{dir = 1},/turf/closed/shuttle/dropship1/transparent{icon_state = "84"},/area/shuttle/drop1/sulaco)
+"aqJ" = (/obj/machinery/computer/security/almayer_network,/obj/structure/table/almayer,/obj/structure/window/reinforced/toughened{dir = 4},/obj/structure/window/reinforced/toughened,/turf/open/floor/plating/almayer,/area/almayer/command/cic)
```
Comment (Rohesie) [inline maps/backup/Z.03.USS_Almayer.dmm]: These changes seem unrelated. What are they about?

Comment (Ghommie) [reply maps/backup/Z.03.USS_Almayer.dmm]: It's the backup file being overwritten by a newer one, forgot to check them out from the commited files. I'll revert.
